x2gbfs/providers/deer.py

```diff
@@ -168,6 +168,7 @@ def load_stations(self, default_last_reported: int) -> Tuple[Dict, Dict]:
                 '_city': elem.get('city'),  # Non-standard
                 'rental_methods': ['key'],
                 'is_charging_station': True,
+                'capacity': 2,  # not provided by API
```

Comment:
Could you give an explanation why 2 is the appropriate number and not 1 or anything else?

Reply:
Each station should have a capacity, so that we can track the overall number of carsharing cars in Baden-Württemberg.
Deer told us about their capacity:
_ich habe gerade nochmal Rücksprache mit unserem Carsharing Team gehalten. Leider ist die Information nicht in dieser Form in unserem System hinterlegt. Da aktuell ein Fahrzeug auch zurückgegeben werden kann, obwohl eine Station vollständig belegt ist, spielt die Maximalkapazität zumindest bei der Rückgabe keine Rolle. 
Alternativ können sich Nutzer zur Routenplanung einen Überblick verschaffen, indem Sie Stationen ohne freie Fahrzeuge mitbetrachten._
Reinhard suggested:
_Im Prinzip könnten wir bei denen die Kapazität auf zwei setzen, wobei an einer Station auch mal mehr Fahrzeuge stehen können. Spannend bei deer: Die Autos sind ja nicht nur Roundtrip-Fähig, sondern erlauben auch A-B-Ausleihe von Station zu Station._